doc/make.jl

```diff
@@ -40,16 +40,16 @@ cd(joinpath(@__DIR__, "src")) do
     end
 end
 
-# Generate a suitable markdown file from NEWS.md and put it in src
-str = read(joinpath(@__DIR__, "..", "NEWS.md"), String)
-splitted = split(str, "<!--- generated by NEWS-update.jl: -->")
-@assert length(splitted) == 2
-replaced_links = replace(splitted[1], r"\[\#([0-9]*?)\]" => s"[#\g<1>](https://github.com/JuliaLang/julia/issues/\g<1>)")
-write(joinpath(@__DIR__, "src", "NEWS.md"), replaced_links)
+# # Generate a suitable markdown file from NEWS.md and put it in src
```

Comment:
Commenting out this for now since it is completely empty, and it looks horrible.